go/pkg/testutil/containers.go

```
@@ -48,14 +48,24 @@ func (c *Containers) RunMySQL() string {
+	cfg.Passwd = "password"
+	cfg.Net = "tcp"
+	cfg.Addr = fmt.Sprintf("localhost:%s", resource.GetPort("3306/tcp"))
+	cfg.DBName = "unkey"
+	cfg.ParseTime = true
+	cfg.Logger = &mysql.NopLogger{}
 
 	var db *sql.DB
 	require.NoError(c.t, c.pool.Retry(func() error {
-		db, err = sql.Open("mysql", addr)
+
+		connector, err := mysql.NewConnector(cfg)
```

golangci-lint (1.62.2)

shadow: declaration of "err" shadows declaration at line 39

(govet)